argo/providers/go.mod

```diff
@@ -3,96 +3,152 @@ module github.com/sky-uk/kfp-operator/argo/providers
 go 1.20
 
 require (
-	cloud.google.com/go/aiplatform v1.26.0
-	cloud.google.com/go/pubsub v1.9.0
-	cloud.google.com/go/scheduler v1.7.0
-	cloud.google.com/go/storage v1.27.0
+	cloud.google.com/go/aiplatform v1.48.0
+	cloud.google.com/go/pubsub v1.33.0
+	cloud.google.com/go/scheduler v1.10.1
+	cloud.google.com/go/storage v1.30.1
+	github.com/argoproj/argo-events v1.8.1
```

Comment:
I imagine this is the line that really changed. I wish Go didn't have such an awful dependency management system.

Reply:
that is correct